From b01ff793c5bb08c1cf9deb14b3d486ebf8c2e904 Mon Sep 17 00:00:00 2001
From: Kei Okada <kei.okada@gmail.com>
Date: Wed, 4 Nov 2015 21:09:13 +0900
Subject: [PATCH 3/3] need check if jade

---
 CMakeLists.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 355901b..2a19a43 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,7 +2,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(euscollada)
 
-IF($ENV{ROS_DISTRO} STREQUAL "indigo")
+IF($ENV{ROS_DISTRO} STREQUAL "jade")
 find_package(catkin REQUIRED COMPONENTS collada_urdf rospack collada_parser resource_retriever)
 ELSE()
 find_package(catkin REQUIRED COMPONENTS collada_urdf rospack collada_parser urdfdom resource_retriever)
@@ -13,7 +13,7 @@ catkin_package()
 set(ENV{PKG_CONFIG_PATH} "$ENV{PKG_CONFIG_PATH}:${CATKIN_DEVEL_PREFIX}/lib/pkgconfig")
 
 find_package(PkgConfig)
-IF($ENV{ROS_DISTRO} STREQUAL "indigo")
+IF($ENV{ROS_DISTRO} STREQUAL "jade")
 pkg_check_modules(urdf_dom urdfdom REQUIRED)
 ENDIF()
 pkg_check_modules(colladadom collada-dom-150 REQUIRED)
-- 
1.7.9.5

